.github/workflows/main.yml

```diff
@@ -0,0 +1,15 @@
+on:
```

Comment:
This is an example of it running on this PR https://github.com/forcedotcom/sfdx-scanner/pull/152/checks?check_run_id=947288695